lib/galaxy/dependencies/pipfiles/update.sh

```diff
@@ -32,13 +32,14 @@ done
 THIS_DIRECTORY="$(cd "$(dirname "$0")" > /dev/null && pwd)"
 ENVS="develop
 flake8
-flake8_imports"
+flake8_imports
+default"
 
 for env in $ENVS
 do
         cd "$THIS_DIRECTORY/$env"
-        pipenv lock
-        pipenv lock -r > pinned-hashed-requirements.txt
+        pipenv lock --two
+        pipenv  --bare lock --two -r | sed 's/^Using.*$//' > pinned-hashed-requirements.txt
```

Comment:
Is the `sed` for deleting the first line? It's actually leaving a first empty line, I think that `sed '1d'` should work better.